source/products/roland/getsafebaserate/rest-api.rst

```diff
@@ -176,11 +176,11 @@ Example Request
 
     curl https://api.kasko.io/payments \
         -X POST \
-        -u <YOUR SECRET API KEY>: \
+        -u YOUR SECRET API KEY: \
```

Comment:
Why removed <> ?

Reply:
Quality of life feature to make it easier to copy/paste the example request ids. So far our docs don't have a consistent rule for this, therefore I chose this style.